deps/rabbit/src/rabbit_stream_queue.erl

```diff
@@ -1039,3 +1041,12 @@ set_leader_pid(Pid, QName) ->
 close_log(undefined) -> ok;
 close_log(Log) ->
     osiris_log:close(Log).
+
+-spec list_with_minimum_quorum() -> [amqqueue:amqqueue()].
+list_with_minimum_quorum() ->
+    lists:filter(fun (Q) ->
+                         StreamId = maps:get(name, amqqueue:get_type_state(Q)),
+                         {ok, Members} = rabbit_stream_coordinator:members(StreamId),
```

Comment:
may be worth commenting that this function returns the coordinator state which may well be a little bit stale. That said this calculation always risks being stale so in practice it doesn't really matter :)